core/src/main/java/org/elasticsearch/index/mapper/core/DateFieldMapper.java

```diff
@@ -394,17 +394,21 @@ public long parseToMilliseconds(Object value, boolean roundUp,
         @Override
         public FieldStats.Date stats(IndexReader reader) throws IOException {
             String field = name();
-            long size = PointValues.size(reader, field);
-            if (size == 0) {
-                return null;
+            long size;
+            try {
+                size = PointValues.size(reader, field);
+                if (size == 0) {
+                    return new FieldStats.Date(reader.maxDoc(), isSearchable(), isAggregatable(), dateTimeFormatter());
+                }
+            } catch (Exception e) {
```

Comment:
I am a bit worried about this but on the other hand I don't think this change should be put on hold until the bug is fixed. Maybe we should remove the catch block and mark failing tests with `@AwaitsFix` and a link to https://issues.apache.org/jira/browse/LUCENE-7257 ?